### PR TITLE
Add fixes to combustion and add pr fixes

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

add fixes to combustion and PR fixes

### CODE REVIEW

File: backend_modules/libvirt/host/combustion [hunk 1]
```diff
@@ -14,38 +14,49 @@ systemctl enable sshd.service
 ${ gpg_keys }
 
 # Add repositories
-zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/repositories/SUSE:/CA/15.5/ ca_suse
+zypper ar http://${ user_mirror_image ? mirror : "download.opensuse.org" }/repositories/SUSE:/CA/15.5/ ca_suse
```
Comment (cbosdo) [inline backend_modules/libvirt/host/combustion]: No, this looks like a typo, the value in the template is `use_mirror_image`. Where does the 'r' come from?

File: backend_modules/libvirt/host/combustion [hunk 2]
```diff
@@ -14,38 +14,49 @@ systemctl enable sshd.service
...
 %{ if container_server || container_proxy }
-zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/openSUSE_Leap_Micro_5.5/ container_utils
+zypper ar http://${ user_mirror_image ? mirror : "download.opensuse.org" }/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/openSUSE_Leap_Micro_5.5/ container_utils
```
Comment (cbosdo) [inline backend_modules/libvirt/host/combustion]: Here too

File: backend_modules/libvirt/host/combustion [hunk 3]
```diff
@@ -14,38 +14,49 @@ systemctl enable sshd.service
...
 %{ endif }
 
 %{ if testsuite }
-zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/repositories/systemsmanagement:/Uyuni:/Test-Packages:/Pool/rpm/ test_packages_pool
+zypper ar http://${ user_mirror_image ? mirror : "download.opensuse.org" }/repositories/systemsmanagement:/Uyuni:/Test-Packages:/Pool/rpm/ test_packages_pool
```
Comment (cbosdo) [inline backend_modules/libvirt/host/combustion]: Here too

Comment (jordimassaguerpla) [reply backend_modules/libvirt/host/combustion]: Thanks! I just fixed it in the next commit

**Comment by jordimassaguerpla** [comment]
@cbosdo I have rebased

**Pull request merged**